index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet" />
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <section class="headersec">
+    <div class="topnav-logo">
+      <h4>My Logo</h4>
+    </div>
+    <div class="menu-icon-wrapper">
+      <div class="menu-icon" onclick="toggleCssMenu(this)">
+        <div class="three-line">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </div>
+    </div>
+    <div id="cssmenu">
+
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">Creative</a></li>
+        <li><a href="#">Studio</a></li>
+        <li><a href="#">Press</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </div>
+
+    <section class="about-flex-container">
+      <div class="name">
+        <h1>I’m Lilian,
+          Glad to see you!</h1>
+      </div>
```

Comment:
- [ ] The section should have a  proper background image (decorations are always background images, not `img` tags)